### ISSUE TITLE
GRU-LSTM 

### ISSUE THREAD

**Issue by UM-Titan** [ElizabethEwurabena/CV_ENG_8160#2]
1. Don't evaluate on the same training data. Split data into training and testing. 
2. Shuffle training and not testing. The dataloader has sort our items so you can shuffle. 
3. MyGRU and MyLSTM models not uploaded. 
4. different horizons not evaluated .
